### DISCUSSION TITLE
When it comes to config keys, do they have a default value if they are not present in the config file?

### DISCUSSION THREAD

**Discussion by SebbeJohansson** [Taxel/PlexTraktSync#1812] [Support] [answered]
Hi! If i dont have a key-value pair inside of the config file, do they default to something? Maybe what ever is in the default config file? I tried looking in both the documentation, PRs and in discussions but couldn't find an answer.

**Comment by glensc** [comment]
what config file you refer to? can't read your mind or display

**Reply by SebbeJohansson** [reply]
Both `config.yml` and `server.yml`. I guess they "fallback" to what ever is in for example `config.default.yml`?

**Reply by glensc** [reply, marked as answer]
config.yml is merged with config.default.yml, server.yml has no fallback, and there's also .env file

additionally, sync section is merged with server config from server.yml (see readme)

**Reply by glensc** [reply]
you can see parsed config with "config" subcommand, but it doesn't yet display result when merging with server sync config. so if you don't have "sync" section in servers.yml it would be accurate.
